Source/CarthageKit/Git.swift

```diff
@@ -68,12 +68,7 @@ public struct GitURL: Equatable {
 
 	/// Strips any trailing .git in the given name, if one exists.
 	private func stripGitSuffix(string: String) -> String {
-		if string.hasSuffix(".git") {
-			let nsString = string as NSString
-			return nsString.substringToIndex(nsString.length - 4) as String
-		} else {
-			return string
-		}
+		return string.stringByReplacingOccurrencesOfString(".git", withString: "")
```

Comment:
This will replace all occurrences of `.git`, which is definitely not what we want. I could have a repo named `i.love.github`, and this would change it to `i.lovehub`.